acsoo/templates/project/+project.name+/.gitlab-ci.yml.bob

```diff
@@ -43,12 +57,11 @@ flake8:
 {{% endif %}}
 test:
   stage: test
-  tags:
-    - odoo-{{{ odoo.series }}}
   script:
     - virtualenv --python=$(which {{{ python_version }}}) venv
     - venv/bin/pip install coverage
-    - venv/bin/pip install -r requirements.txt -e .
+    - venv/bin/pip install --no-deps --no-index release/*.whl
+    - venv/bin/pip install -e . --no-deps --no-index
```

Comment:
`venv/bin/pip install --no-deps --no-index -e . ` for symmetry

Plus add a comment "pip install project in editable mode so coverage sees it"

Reply:
or "Although the project is part of *.whl, install it in editable mode so coverage sees it"